examples/calculations/submit_geomopt.py

```diff
@@ -0,0 +1,184 @@
+"""Example code for submitting single point calculation"""
+
+from pathlib import Path
+from typing import Union
+
+from ase.build import bulk
+from ase.io import read
+import click
+
+from aiida.common import NotExistent
+from aiida.engine import run_get_node
+from aiida.orm import Bool, Float, Str, StructureData, load_code, load_node
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.model import ModelData
+
+
+def load_model(string: Union[str, Path, None], architecture: str) -> ModelData:
```

Comment:
`string` -> `model_path` or something more meaningful?

Reply:
yeah I didn't know how to call it cause it could be a url or a path